Build is failing;

```
daemon/cluster/cluster.go:1::warning: file is not goimported (goimports)
16:22:28 daemon/cluster/swarm.go:1::warning: file is not goimported (goimports)
16:22:28 daemon/cluster/swarm.go:457:4:warning: undeclared name: grpc (gosimple)
16:22:28 daemon/cluster/cluster.go:407:3:warning: undeclared name: grpc (unconvert)
16:22:28 daemon/cluster/swarm.go:457:4:warning: undeclared name: grpc (unconvert)
16:22:30 Build step 'Execute shell' marked build as failure

--- a/daemon/cluster/cluster.go
+++ b/daemon/cluster/cluster.go
@@ -58,6 +58,7 @@ import (
 	swarmnode "github.com/docker/swarmkit/node"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
 )
 
 const swarmDirName = "swarm"
@@ -402,7 +403,10 @@ func (c *Cluster) Cleanup() {
 func managerStats(client swarmapi.ControlClient, currentNodeID string) (current bool, reachable int, unreachable int, err error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
-	nodes, err := client.ListNodes(ctx, &swarmapi.ListNodesRequest{})
+	nodes, err := client.ListNodes(
+		ctx, &swarmapi.ListNodesRequest{},
+		grpc.MaxCallRecvMsgSize(defaultRecvSizeForListResponse),
+	)
 	if err != nil {
 		return false, 0, 0, err
 	}

--- a/daemon/cluster/swarm.go
+++ b/daemon/cluster/swarm.go
@@ -19,6 +19,7 @@ import (
 	swarmnode "github.com/docker/swarmkit/node"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
 )
 
 // Init initializes new cluster from user provided request.
@@ -452,7 +453,10 @@ func (c *Cluster) Info() types.Info {
 
 		info.Cluster = &swarm.ClusterInfo
 
-		if r, err := state.controlClient.ListNodes(ctx, &swarmapi.ListNodesRequest{}); err != nil {
+		if r, err := state.controlClient.ListNodes(
+			ctx, &swarmapi.ListNodesRequest{},
+			grpc.MaxCallRecvMsgSize(defaultRecvSizeForListResponse),
+		); err != nil {
 			info.Error = err.Error()
 		} else {
 			info.Nodes = len(r.Nodes)

